### DISCUSSION TITLE
Welcome to F1-API-JSON Discussions!

### DISCUSSION THREAD

**Discussion by yashkathe** [yashkathe/F1-API#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tarunbharadwaj** [comment]
Hi @yashkathe I came across your F1 node package, but I'm not sure if it can be used in nodejs application, would you like to understand some sample code on how to using this package? I primarily want to create a web application using angular to display latest rankings/scores of drivers

**Reply by yashkathe** [reply]
just download the package and have a look at the readme

```node
const f1Api = require('f1-api-node') 

const myFunction = async () => {
    const driverLineup = await f1Api.getDriverData()
    console.log(driverLineup)
}

myFunction()
```
in place of MyFunction enter the name of the function you want to use  
some functions require extra parameters  
read the readme, I have documented every function

P.S: Open a new discussion or issue if you run into an issue
